src/Package/Fetch.zig

```diff
@@ -274,14 +274,19 @@ pub const Location = union(enum) {
     remote: Remote,
     /// A directory found inside the parent package.
     relative_path: Cache.Path,
-    /// Recursive Fetch tasks will never use this Location, but it may be
-    /// passed in by the CLI. Indicates the file contents here should be copied
-    /// into the global package cache. It may be a file relative to the cwd or
-    /// absolute, in which case it should be treated exactly like a `file://`
-    /// URL, or a directory, in which case it should be treated as an
-    /// already-unpacked directory (but still needs to be copied into the
-    /// global package cache and have inclusion rules applied).
-    path_or_url: []const u8,
+    /// A URL passed via the CLI.
+    /// Indicates that the archive the URL refers to should
+    /// be fetched and copied into the global package cache.
+    /// Recursive Fetch tasks will never use this Location.
+    cli_url: std.Uri,
```

Comment:
src/Package/Fetch.zig shouldn't know about the CLI.

Reply:
Please clarify, is your objection just to the names having "cli" in them, or that there are fields that are only used to serve the needs of `zig fetch` and thus only used by the CLi in practice? If its the names they can be changed, no problem, but if its the latter I don't really see a good way around it.

Reply:
Renamed them to just `url` and `path`, do you think that's better? Any other possible problems were present with the old `url_or_path` as well, all I did was break them out into two distinct fields.